### ISSUE TITLE
Java/Minecraft settings tab does not fit all settings on screen when field has a long value

### ISSUE THREAD

**Issue by Aphyxia** [ATLauncher/ATLauncher#963]
### What happened?

Java/Minecraft settings tab does not fit on the screen with many Java parameters. There is also no scroll functionality. I have to remove parameters to see the other settings.
![image](https://github.com/user-attachments/assets/17d9004f-4b5a-4c22-af24-c0b585f4288c)

### Operating System

Windows

### Is this actually a launcher bug?

- [X] I am sure this is a bug with the launcher and not an issue with running Minecraft

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by RyanTheAllmighty** [comment]
Yeah the box should be fixed size and scrollable, not like that. That's not great

**Comment by Doomsdayrs** [comment]
Is no one wondering why there are so many configurations?

---

Seems its not in a scroll box, will add
